src/app/features/courses/courses-list/courses-list.component.html

```diff
@@ -1,7 +1,21 @@
-<div *ngFor="let course of courses">
+<div *ngFor="let course of courses$ | async">
   <app-course-card [courseObject]="course" [isEditable]="editable">
-    <app-button courseShow [buttonText]="buttonLabelService.getLabel('SHOW_COURSE')"></app-button>
-    <app-button courseEdit [iconName]="buttonLabelService.getLabel('TRASH_CAN')"></app-button>
-    <app-button courseDelete [iconName]="buttonLabelService.getLabel('PENCIL')"></app-button>
+    <app-button
+      courseShow
+      [buttonText]="buttonLabelService.getLabel('SHOW_COURSE')"
+      (clicked)="router.navigate(['/courses', course.id])"
```

Comment:
I'd prefer to handle the navigation in the controller layer instead of the view.
`(click)="handleClick(course.id)"`